projects/fw_102/src/main.c

```diff
@@ -0,0 +1,42 @@
+#include <stdint.h>  // for integer types
+
+#include "interrupt.h"   // interrupts are required for soft timers
+#include "log.h"         // for printing
+#include "soft_timer.h"  // for soft timers
+#include "wait.h"        // for wait function
+
+#define COUNTER_PERIOD_MS 500  // ms
```

Comment:
Nit: you don't need to specify the units in a comment since it's already in the constant name